### ISSUE TITLE
[Hand4Whole]I would like to ask a question about the Agora dataset.

### ISSUE THREAD

**Issue by cyk990422** [mks0601/Hand4Whole_RELEASE#59]
Hello Dr. Moon.
I would like to ask if you would mind sharing the yolov5 detector that you finetune on agora. I would like to use it to try the detection effect on the synthetic data set of Agora.

**Comment by mks0601** [comment]
sorry I do not have it, but you can easily fine-tune it.

**Comment by cyk990422** [comment]
Thanks for your reply, nevermind, I fine-tuned one myself.
Sorry, I have another question to ask you, that is, the Agora data set contains some labels of kids. The betas of these data contain 11 parameters. Did you only take the first 10 parameters of the betas for training?

**Comment by mks0601** [comment]
I think so. I did not take care much about kids...

**Comment by cyk990422** [comment]
Thank you very much for your patient reply!
